src/components/SettingsButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
```

Comment:
Já que você usou o Link, vc precisa importá-lo logo após o React.
```suggestion
import { Link } from 'react-router-dom'
```